QCircLearning/utils.py

```diff
@@ -49,9 +49,9 @@ def __init__(self, patience=5, min_delta=0.0, verbose=False):
         self.verbose = verbose
         self.best_loss = None
         self.counter = 0
-        self.early_stop = False
+        # self.early_stop = False
 
-    def __call__(self, current_loss):
+    def __call__(self, current_loss, stop=False):
```

Comment:
why there will be a stop=False ?

Reply:
I want make sure it is not True at beginning

Reply:
There is no such scenario that needs to be checked True or false at the beginning